### PR TITLE
CU-868ay3v6d - Update pinnacleqi/coding-standard to use PHP 8.3

### PR DESCRIPTION
**Pull request by krist4lle**
Updated PHP version in composer
image

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,13 +4,18 @@
     "type": "phpcodesniffer-standard",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4 || 8.0.* || 8.1.*",
+        "php": "^7.4 || 8.0.* || 8.1.* || 8.2.* || 8.3.*",
```
Comment (ChrisSchwerdt) [inline composer.json]: I think we can be more flexible and just allow PHP 7.4 and PHP 8.x.
```suggestion
        "php": "^7.4 || ^8.0",
```

File: composer.json [hunk 2]
```diff
@@ -4,13 +4,18 @@
...
         "squizlabs/php_codesniffer": ">=3.5"
     },
     "autoload": {
         "psr-4": {
             "PinnacleCodingStandard\\": "PinnacleCodingStandard"
         }
+    },
+    "config": {
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
```
Comment (ChrisSchwerdt) [inline composer.json]: I don't think we need this configuration at the library level.

**Review by ChrisSchwerdt — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (krist4lle) [reply composer.json]: I removed it once. It returned XD

Comment (ChrisSchwerdt) [reply composer.json]: Okay, that's fine then.

**Review by ChrisSchwerdt — APPROVED** [review]

**Pull request merged**